### DISCUSSION TITLE
Rapport PDF

### DISCUSSION THREAD

**Discussion by ghost** [dbarzin/mercator#249] [Q&A]
Bonjour,

Est-il possible avec votre outils de cartographie de générer un rapport en pdf (ou autre format de texte) un résumé de la vue global d'exploration? Un document qui regrouperait les rapports qu'on peut avoir dans les listes des tables?

Cordialement,

RODRIGUEZ Erwan

**Comment by dbarzin** [comment]
Bonsoir,
Il y a un "rapport de la cartographie du système d'information" qui se trouve dans le menu "documentation" -> " rapports".
Ce document reprend l'ensemble de la cartographie du système d'information et devrait couvrir vos besoins.
Si ce n'est pas le cas, peux-tu être plus précis ?
Merci,
Didier

**Reply by ghost** [reply]
Ah oui merci, c'est tout a fait ce que je cherchais. Je n'y avais pas fait attention désolé. Merci beaucoup !
